### ISSUE TITLE
Feedback course 07: Metadata

### ISSUE THREAD

**Issue by christian-rli** [OpenEnergyPlatform/examples#187]
Please review course 07 : https://openenergyplatform.github.io/academy/courses/07_metadata/
and provide your feedback as comments below. Feedback can include pointing out errors (such as broken links, things that do not work as described) any suggestions for improvements, things that you are expecting from such a course but find they are missing, etc.

In your comment, please make sure to reference the link to the material you're referring to (the courses link to various materials)

**Comment by han-f** [comment]
I read through the course today and wonder whether you could - additionally to all the links -  also provide a hands-on example for someone who has never worked with metadata and looks for how to do it right away. An idea could be to show a table with the field names of the OEMetadatastandard on the left, the description what it should hold in the middle, and an example on the right. This already exists [here](https://github.com/OpenEnergyPlatform/oemetadata/blob/master/metadata/latest/metadata_key_description.md#metadata-keys-with-a-description-and-example ) and it may be helpful to display this directly in the course with some context information. 

For some users some more context information around these table may also be helpful, e.g. I am not sure if everyone knows without further text that under section [Licence keys](https://github.com/OpenEnergyPlatform/oemetadata/blob/master/metadata/latest/metadata_key_description.md#license-keys) _An array of objects of the license(s) under which the described package is provided. It can depend on the licenses of the sources (copyleft or share-alike) or can be granted by the creator of the data_ refers to their own published dataset. A simple inftroductory set for each section and extra info for some columns may help in such cases, for example by letting the reader know that the _described package_ refers to the data he/she provided. . 
Happy to check this also together.

**Comment by flohrreija-oeko** [comment]
The link "Validate that your metadata is a datapackage" leads to the tutorial Introduction to OpenEnergyMetadata (OEM) (see issue #225). I assume it refers to the section "Compatible with frictionless data" in this tutorial - to clarify this it could be useful to include the word frictionless in the link, e.g. "How to validate that you metadata is a datapackage compatible with frictionless data" (maybe this is not what the link referred to and I simply missunderstood it)

**Comment by flohrreija-oeko** [comment]
The first and third of the supplementary materials in the last section of the course are not linked - maybe because not published yet? If published already the link should be included.

**Comment by jh-RLI** [comment]
> The link "Validate that your metadata is a datapackage" leads to the tutorial Introduction to OpenEnergyMetadata (OEM) (see issue #225). I assume it refers to the section "Compatible with frictionless data" in this tutorial - to clarify this it could be useful to include the word frictionless in the link, e.g. "How to validate that you metadata is a datapackage compatible with frictionless data" (maybe this is not what the link referred to and I simply missunderstood it)

I agree it should link to the "frictionless datapackae" section and also include the working.  

Should i update this?

**Comment by jh-RLI** [comment]
> The first and third of the supplementary materials in the last section of the course are not linked - maybe because not published yet? If published already the link should be included.

Are you referring to this section? The links work for me :)
![image](https://github.com/user-attachments/assets/15b6d7d5-fb28-4277-827c-1aee20e593d8)

Edit:
Ah sorry now i got it the linked documents on zenodo are not available. Hm i will ask around what happend there

Edit:
Seems like this is only a display error on zenodo?

**Comment by flohrreija-oeko** [comment]
> > The first and third of the supplementary materials in the last section of the course are not linked - maybe because not published yet? If published already the link should be included.
> 
> Are you referring to this section? The links work for me :) ![image](https://private-user-images.githubusercontent.com/38939526/349468558-15b6d7d5-fb28-4277-827c-1aee20e593d8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEyMTc4NTMsIm5iZiI6MTcyMTIxNzU1MywicGF0aCI6Ii8zODkzOTUyNi8zNDk0Njg1NTgtMTViNmQ3ZDUtZmIyOC00Mjc3LTgyN2MtMWFlZTIwZTU5M2Q4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE3VDExNTkxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgxYThhNTU4Njc4ZjRkMTZiNTBiZTFjYWQwOTk3NmU2MmQ0ZDc0ZDQyZWQzNjYwY2RkMDdhYWM2ZTU5OWQzMjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1mIg_7lyO577auz2FA_A25Q7WOju4T7ziftgq1jPbo0)
> 
> Edit: Ah sorry now i got it the linked documents on zenodo are not available. Hm i will ask around what happend there
> 
> Edit: Seems like this is only a display error on zenodo?

I missunderstood it and thought there should be 4 materials (for 4 bulletpoints) linked and that the first and third one had no link - i did not realize it was just the titles of the linked material, my mistake! I also get the display error on zenodo now but could acces the presentation 2 days ago

**Comment by Ludee** [comment]
Update DALIA metadata
